lxc/main.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```

Comment:
This change shouldn't be included. It's most likely your editor deciding to expand the package name for some reason.